frame/session/src/lib.rs

```diff
@@ -607,6 +620,10 @@ impl<T: Config> Module<T> {
 		let (next_validators, next_identities_changed)
 			= if let Some(validators) = maybe_next_validators
 		{
+			let validators = BoundedVec::<T::ValidatorId, T::MaxValidators>::truncating_from(
+				validators,
+				Some("Session Rotate Session Maybe Next Validators"),
+			);
```

Comment:
Wouldn't it make sense that `SessionManager` is also parameterized on `MaxValidators` and `SessionManager::new_session` returns a `BoundedVec` instead?